InstantSearch.js/routing-seo-friendly/src/search-router.js

```diff
@@ -50,38 +47,35 @@ const router = instantsearch.routers.history({
     const routeStateString = location.pathname.split('/')[1];
     const category = getCategoryName(routeStateString.split('/')[0]);
     const queryParameters = qsModule.parse(location.search.slice(1));
-    const { q = '', p, brands } = queryParameters;
+    const { query = '', page, brands } = queryParameters;
 
     return {
-      q: decodeURIComponent(q),
-      p,
+      query: decodeURIComponent(query),
+      page,
       category: decodeURIComponent(category),
-      brands: decodeURIComponent(brands),
+      brands: brands && brands.map(decodeURIComponent),
```

Comment:
We could use `[]` otherwise an empty value is selected when brands is without value.

> https://yjojj4lnrz.codesandbox.io/search?query=apple&brands